pkg/ingester/client/timeseries_test.go

```diff
@@ -0,0 +1,26 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLabelAdapter_Marshal(t *testing.T) {
+	tests := []struct {
+		bs *LabelAdapter
+	}{
+		{&LabelAdapter{Name: "foo", Value: "bar"}},
+		{&LabelAdapter{Name: "very long label name", Value: "very long label value"}},
+		{&LabelAdapter{}},
```

Comment:
This case can't occur for real, can it?
The case of nonblank Name and blank Value is possible, if you're aiming to cover all cases.

Reply:
Ha yes true forgot about that one ! This all empty can't really occur but I figure it should still works ok.